latex2sympy.py

```diff
@@ -484,22 +487,18 @@ def convert_atom(atom):
         else:
             symbol = sympy.Symbol(symbol_name, real=True)
 
+        if is_percent:
+            return sympy.Mul(symbol, sympy.Pow(100, -1, evaluate=False), evaluate=False)
```

Comment:
@builtbywill Any reason to not make both of them rational? I'm not sure what the difference is exactly for changing the number percents below.

Reply:
@tragessere I did try to do that, but `Symbol`s are not valid args for `Rational`. The difference is the numbers below, the first arg is a Number or Rational, which are valid args.